apps/challenges/migrations/0091_auto_manual_approval.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.20 on 2023-06-09 12:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('participants', '0012_remove_docker_repository_uri_from_team'),
+        ('challenges', '0090_challenge_allow_resuming_submissions'),
```

Comment:
You will need to update the migration here. Delete this and generate migration again.